### PR TITLE
Support arm64 builds

### PR DESCRIPTION
**Pull request by jalaziz**
Update the goreleaesr config with:
* arm64 binary builds
* mutli-arch docker image builds

Also, add the newly required dependencies for multi-arch builds to the
Github Actions workflows.

### CODE REVIEW

**Comment by jalaziz** [comment]
I've testing the workflow changes on my fork. You can download and test the multi-arch image here: https://github.com/jalaziz/flipt/pkgs/container/flipt

**Review by markphelps — APPROVED** [review]
lgtm! thank you @jalaziz! just two minor typos

**Comment by markphelps** [comment]
@all-contributors please add @jalaziz for platform

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -25,6 +25,14 @@ jobs:
 
       - uses: arduino/setup-task@v1
 
+      - uses: docker/setup-qemu-action@v2
+
+      - uses: docker/setup-buildx-action@v2
+
+      - name: Intsall GCC for arm64
```
[the remark this answers is not in the document]
Comment (jalaziz) [reply .github/workflows/release.yml]: Fixed!

File: .github/workflows/snapshot.yml [hunk 2]
```diff
@@ -18,6 +18,14 @@ jobs:
 
       - uses: arduino/setup-task@v1
 
+      - uses: docker/setup-qemu-action@v2
+
+      - uses: docker/setup-buildx-action@v2
+
+      - name: Intsall GCC for arm64
```
[the remark this answers is not in the document]
Comment (jalaziz) [reply .github/workflows/snapshot.yml]: Fixed!

**Pull request merged**